addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2081,6 +2081,74 @@ QUnit.test('chat window: chat header should not be clickable when thread has mul
     );
 });
 
+QUnit.test('2 chat windows: check shift operations are well entitled with a LTR locale', async function (assert) {
+    assert.expect(2);
+
+    // 2 channels are expected to be found in the messaging menu
+    // only their existence matters, data are irrelevant
+    this.data['mail.channel'].records.push({}, {});
+    await this.start();
+
+    await afterNextRender(() => {
+        document.querySelector('.o_MessagingMenu_toggler').click();
+    });
+    await afterNextRender(() => {
+        document.querySelectorAll('.o_MessagingMenu_dropdownMenu .o_NotificationList_preview')[0].click();
+    });
+    await afterNextRender(() => {
+        document.querySelector('.o_MessagingMenu_toggler').click();
+    });
+    await afterNextRender(() => {
+        document.querySelectorAll('.o_MessagingMenu_dropdownMenu .o_NotificationList_preview')[1].click();
+    });
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftPrev').title,
+        "Shift left",
+        "shift previous operation should be entitled 'Shift left' as locale is ltr"
+    );
+
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftNext').title,
+        "Shift right",
+        "shift previous operation should be entitled 'Shift right' as locale is ltr"
+    );
+});
+
+QUnit.test('2 chat windows: check shift operations are well entitled with a RTL locale', async function (assert) {
+    assert.expect(2);
+
+    // 2 channels are expected to be found in the messaging menu
+    // only their existence matters, data are irrelevant
+    this.data['mail.channel'].records.push({}, {});
+    await this.start();
+    this.env.messaging.locale.update({textDirection: 'rtl'});
+
+    await afterNextRender(() => {
+        document.querySelector('.o_MessagingMenu_toggler').click();
+    });
+    await afterNextRender(() => {
+        document.querySelectorAll('.o_MessagingMenu_dropdownMenu .o_NotificationList_preview')[0].click();
+    });
+    await afterNextRender(() => {
+        document.querySelector('.o_MessagingMenu_toggler').click();
+    });
+    await afterNextRender(() => {
+        document.querySelectorAll('.o_MessagingMenu_dropdownMenu .o_NotificationList_preview')[1].click();
+    });
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftPrev').title,
+        "Shift right",
+        "shift previous operation should be entitled 'Shift right' as locale is rtl"
+    );
+
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftNext').title,
+        "Shift left",
+        "shift previous operation should be entitled 'Shift left' as locale is rtl"
+    );
+    this.env.messaging.locale.update({textDirection: 'ltr'});
```

Comment:
is this needed? why?

Reply:
Forgot to remove it

Reply:
In fact this text only works if the language of the browser is rtl. How can I fool the test env to add o_rtl class on body and to make this information reflected in the messaging store ?

Reply:
For models you should be able to override `_t` from `env` at `start()`, but I'm not sure if this is enough for the class in `body`. I guess do it manually if you must?